cst438_register/src/main/java/com/cst438/domain/StudentDTO.java

```diff
@@ -0,0 +1,45 @@
+package com.cst438.domain;
+
+import java.util.Objects;
+
+public class StudentDTO {

```

Comment:
I did not have to create a StudnetDTO for my class, but this gives a clear idea of how to create a controller from scratch.